libraries/wavemap/include/wavemap/utils/time.h

```diff
@@ -0,0 +1,17 @@
+#ifndef WAVEMAP_UTILS_TIME_H_
+#define WAVEMAP_UTILS_TIME_H_
+
+#include <chrono>
+
+namespace wavemap {
+using Time = std::chrono::steady_clock;
```

Comment:
I assume steady over high_resolution so you're certain time doesn't run backwards?